server/ot_rpc_test.go

```diff
@@ -0,0 +1,98 @@
+package server
+
+import (
+	"bytes"
+	"crypto/tls"
+	"io"
+	"io/ioutil"
+	"math/rand"
+	"mime"
+	"mime/multipart"
+	"net/http"
+	"net/http/httptest"
+	"net/url"
+	"testing"
+
+	"github.com/livepeer/go-livepeer/common"
+	"github.com/livepeer/go-livepeer/core"
+	"github.com/livepeer/go-livepeer/eth"
+	"github.com/livepeer/go-livepeer/net"
+	"github.com/livepeer/lpms/ffmpeg"
+	"github.com/stretchr/testify/assert"
+)
+
+type stubTranscoder struct {
+	called int
+	fname  string
+}
+
+var testRemoteTranscoderResults = [][]byte{[]byte("body1"), []byte("body2")}
+
+func (st *stubTranscoder) Transcode(fname string, profiles []ffmpeg.VideoProfile) ([][]byte, error) {
+	st.called++
+	st.fname = fname
+	return testRemoteTranscoderResults, nil
+}
+
+func TestRemoteTranscoder(t *testing.T) {
+	httpc := &http.Client{Transport: &http.Transport{TLSClientConfig: &tls.Config{InsecureSkipVerify: true}}}
+	profiles := []ffmpeg.VideoProfile{ffmpeg.P720p60fps16x9, ffmpeg.P144p30fps16x9}
+
+	assert := assert.New(t)
+	assert.Nil(nil)
+	notify := &net.NotifySegment{
+		TaskId:   742,
+		Profiles: common.ProfilesToTranscodeOpts(profiles),
+		Url:      "linktomanifest",
+	}
+	tr := &stubTranscoder{}
+	node := &core.LivepeerNode{
```

Comment:
Probably better to use the NewLivepeerNode, eg `node = NewLivepeerNode(nil, tmpdir, nil)` then set any required fields, eg `node.OrchSecret  = ...`

Also, for the tmpdir, I'd suggest actually making a TempDir so it can be cleaned up properly after the test exists; see the various examples around the other test code, eg

https://github.com/livepeer/go-livepeer/blob/c20c0f699cf598093f9ef05b75a7c5c1c28ec088/core/core_test.go#L78-L82

Reply:
Fixed
`WorkDir` is not used in this test